app/components/product_component.rb

```diff
@@ -40,7 +40,7 @@ def column_value(column)
     when 'on_demand'
       @product.on_demand
     when 'tax_category'
-      @product.tax_category.name
+      @product.variant.tax_category.name
```

Comment:
I don't think that the product has a `variant` method. If I'm right then there's a missing spec to cover this.

Reply:
Hmmm I think this is only used in the product UI v2, which isn't in use?

Reply:
I guess that as well. So it's okay to not write specs but it would still be good to fix it.

Reply:
product UI v2 should be deleted I suppose since we started v3.